test/runtests.jl

```diff
@@ -15,38 +16,39 @@ isCI = "CI" ∈ keys(ENV)
 islinux = Sys.islinux()
 visualtests = !isCI || (isCI && islinux)
 datadir = joinpath(@__DIR__,"data")
+test_device = CUDALibs()
```

Comment:
Try to avoid underscore in variable names, it is not usual in Julia code style. Can you maybe use `device` here?